command/kv_destroy.go

```diff
@@ -0,0 +1,119 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+var _ cli.Command = (*KVDestroyCommand)(nil)
+var _ cli.CommandAutocomplete = (*KVDestroyCommand)(nil)
+
+type KVDestroyCommand struct {
+	*BaseCommand
+
+	flagVersions []string
+}
+
+func (c *KVDestroyCommand) Synopsis() string {
+	return "Destroys a version in the KV store"
```

Comment:
"Destroys" → "Permanently removes"  (What does destroy do? It destroys! 😀)
"a version" → "versions" or "one or more versions" ?